SPECS/ocaml-mirage-profile.spec

```diff
@@ -0,0 +1,58 @@
+%define debug_package %{nil}
+
+Name:           ocaml-mirage-profile
+Version:        0.6.1
+Release:        1%{?dist}
+Summary:        Library for collecting trace data
+License:        BSD-2-clause
+URL:            https://github.com/mirage/mirage-profile/
+Source0:        https://github.com/mirage/mirage-profile/archive/v%{version}/%{name}-%{version}.tar.gz
+BuildRequires:  ocaml
+BuildRequires:  ocaml-ocamldoc
+BuildRequires:  ocaml-camlp4-devel
+BuildRequires:  ocaml-findlib
```

Comment:
This line should not be necessary - ocaml-findlib-devel should require ocaml-findlib